server/src/repositories/media.repository.ts

```diff
@@ -86,6 +86,7 @@ export class MediaRepository implements IMediaRepository {
           rotation: Number.parseInt(`${stream.rotation ?? 0}`),
           isHDR: stream.color_transfer === 'smpte2084' || stream.color_transfer === 'arib-std-b67',
           bitrate: Number.parseInt(stream.bit_rate ?? '0'),
+          holdsThumbnail: (stream.disposition?.attached_pic ?? 0) != 0,
```

Comment:
Wouldn't it be simpler to just filter for streams where this expression is `false`?

Reply:
You could, I was just going for the minimal change that would fix my files, and filtering streams that previously _might_ have worked seemed like there was a remote chance it would break some esoteric video.

Also, if we keep the streams around in our metadata it might make it easier to change the video thumbnail generation later to prioritize using these streams for that purpose.

Reply:
I don't think we need to worry about the former: a video thumbnail stream isn't a video stream, so if that's the only stream it's not going to end well anyway.

To the latter point, that could be interesting, but even then I'd prefer for the thumbnail stream to be separate from the video streams. `probe` could return a `thumbnailStreams` object down the line if we need it.

Reply:
Alright, I've updated the PR to just do stream filtering, which does simplify the changes significantly.

I looked around but I couldn't find any tests of the `probe()` implementation itself (just ones using mocked results from it) so there's no longer an associated unit test for this.